### DISCUSSION TITLE
R 5.1.0 Sunbird CoKreat - Development Phase Sign Off

### DISCUSSION THREAD

**Discussion by Aparna-karri591** [Sunbird-coKreat/Community#37] [General Discussions]
Hello All,

This is Development Phase QA Sign off for the Release 5.1.0.0 of Sunbird CoKreat Building Block.

Environment: Dock Staging
Testing Status: Completed
Defects Dashboard: NA

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1dAlxnsC5A_JvferS_fFx_Quob1ZJbjqcEeMQY6suK9Y/edit#gid=0)

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=12683)

Building Block Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1gqTE5IdfUowcznpci_xKD3wNyePM3LjCRtk7TzAvI4I/edit#gid=1985233391)

cc: @pallakartheekreddy @Nadiya496 @anunagaraj @ShailandraOjha @chitranshu-keshav,@luckynvdu @rajeevsathish @surendrasinghs @sharathkashyap

**Comment by Nadiya496** [comment]
Hello All,

[CO-140](https://project-sunbird.atlassian.net/browse/CO-140) is been added to the scope and its a regression required ticket. This ticket  refers to the CSP changes for the cokreat BB as part of release 5.1.0.

The same is been updated over the development phase Sign Off sheet.

cc: @pallakartheekreddy @Aparna-karri591  @anunagaraj @ShailandraOjha @chitranshu-keshav,@luckynvdu @rajeevsathish @surendrasinghs @sharathkashyap

**Comment by anunagaraj** [comment]
@pallakartheekreddy Please confirm if the sign-off includes the migration scripts and is handed over to the DevOps team.
[@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav Prasad](mailto:keshav.prasad@tarento.com) @G33tha Kindly confirm if you have received the migration scripts to run in the Sunbird ed environment.

CC @luckynvdu @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha

**Comment by anunagaraj** [comment]
@pallakartheekreddy  Please share an update on the above comment as we will not be able to start the regression until we have the confirmation on the migration scripts handed over to DevOps to run in the Sunbird ed environment.

CC @luckynvdu @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha [@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav](mailto:keshav.prasad@tarento.com) @G33tha

**Comment by pallakartheekreddy** [comment]
@anunagaraj From coKreat side, we are ready with the migration script for services which is owned by coKreat BB. Similarly, Ed coKreat also uses Knowlg & inQuiry BBs we need to do integration testing after taking the latest build provided by both  Knowlg & inQuiry BBs. This will take another 3 to 4 days. after that we will provide a signoff for coKreat.
CC @luckynvdu @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha [@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav](mailto:keshav.prasad@tarento.com) @G33tha @Aparna-karri591 Nadiya496

**Comment by sanyapawah** [comment]
@pallakartheekreddy need support to get the PR merged for this use case ([CO-16](https://project-sunbird.atlassian.net/browse/CO-16) from the devops team as requested from [@Santhosh Gandham](mailto:santhosh.g@optit.in)

https://github.com/project-sunbird/sunbird-devops/pull/3685

CC @surendrasinghs

**Reply by anunagaraj** [reply]
@sanyapawah CO-16 has been signed off as part of R5.1.0 and we have also verified this ticket. Can I know what change has been done now?
CC @pallakartheekreddy @chitranshu-keshav @ShailandraOjha @Nadiya496

**Comment by sanyapawah** [comment]
Hi Anusha,

I understand that the test cases presented in the sign off are verified. We
are asking for its next enhancement to be deployed.

No change to the R5.1.0 development.

Thanks

On Thu, Jan 12, 2023 at 11:21 AM Anusha N ***@***.***> wrote:

> @sanyapawah <https://github.com/sanyapawah> CO-16 has been signed off as
> part of R5.1.0 and we have also verified this ticket. Can I know what
> change has been done now?
> CC @pallakartheekreddy <https://github.com/pallakartheekreddy>
> @chitranshu-keshav <https://github.com/chitranshu-keshav> @ShailandraOjha
> <https://github.com/ShailandraOjha> @Nadiya496
> <https://github.com/Nadiya496>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Sunbird-coKreat/discussions/37#discussioncomment-4663919>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A25FHDJE2A4JBKZJBCOYOULWR6LX5ANCNFSM6AAAAAASC7OYEE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
Thanks,

*Sanya Pawah*

Senior Associate | 9549060608

Samagra | Transforming Governance

Address: 9/5 Sarvapriya Vihar, New Delhi-16
***@***.***,77.1994501,17z/data=!3m1!4b1!4m5!3m4!1s0x390ce273981e5dfb:0x7bfceb198af17562!8m2!3d28.5566128!4d77.2016388?shorturl=1>

Website: www.samagragovernance.in

Follow Us: Linkedin
<https://www.linkedin.com/company/samagra-transforming-governance/> |
Youtube <https://www.youtube.com/channel/UCfkXErS-f87xUQkmSKSC8bg> |
Instagram <https://www.instagram.com/samagragovernance/?hl=en>
